pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -356,6 +360,13 @@ func (r *ReconcilePravegaCluster) syncSegmentStoreSize(p *pravegav1alpha1.Praveg
 		if err != nil {
 			return fmt.Errorf("failed to sync pvcs of stateful-set (%s): %v", sts.Name, err)
 		}
+
```

Comment:
Please add unit tests to cover this code.
Also make sure we also have unit tests to check for deletion of PVCs on scale down.

Reply:
added